pipeline/outputs/s3.md

```diff
@@ -49,6 +49,13 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | external\_id                     | Specify an external ID for the STS API, can be used with the role\_arn parameter if your role requires an external ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | None                                    |
+| parquet.compression             | Compression type for parquet. 'uncompressed', 'snappy', 'gzip', 'zstd' are the supported values by default. 'lzo', 'brotli', 'lz4' are not supported for now.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | SNAPPY                                  |
+| parquet.pagesize                | Page size of parquet format. Defaults to 8192 bytes (8KiB).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | 8192                                    |
```

Comment:
Size in bytes right? Let's update that in these fields to be clear.

Reply:
Yup, size should be byte(s) unit. However, page size is a proper noun to specify size of "pagesize" for parquet compression.
ref: https://parquet.apache.org/docs/file-format/configurations/

Reply:
So, do we need to use _Pagesize_ instead of _Page size_ here?

Reply:
Sorry I meant just make it clear the value you enter is in bytes - someone might put in 8Mb or something silly.

Reply:
Ah, got it. I'd confused what you mean. Other parameters which are related of byte(s) unit use bytes in the document. So, I added them.